tests/test_download_core.py

```
@@ -1593,9 +1558,53 @@ def test_process_apk_downloads_enhanced_prerelease_cleanup(tmp_path):
+        "CHECK_APK_PRERELEASES": True,
+    }
+    paths_and_urls = {
+        "android_releases_url": "https://example.invalid/apk",
+        "cache_dir": str(cache_dir),
+        "apks_dir": str(apks_dir),
+    }
+
+    monkeypatch.setattr(
+        downloader,
+        "_get_latest_releases_data",
+        lambda *args, **kwargs: [{"tag_name": "v2.7.8", "prerelease": False}],
```

Ruff (0.14.7)

Unused lambda argument: `args`

(ARG005)